golemhost/filter.go

```diff
@@ -18,20 +19,20 @@ const (
 	FilterComparatorLess
 )
 
-func (filterComparator FilterComparator) ToBinding() binding.GolemApi1_1_6_HostFilterComparator {
+func (filterComparator FilterComparator) ToBinding() host.FilterComparator {
```

Comment:
do we still need all the wrappers? AFAIR many of them were mainly added to hide the not so nice generated names.
there are some with structural differences too (to make them more idiomatic), but probably we should review which are worth to keep, and which can be now used directly

Reply:
this can be a separate round, just wanted to ask, if you considered tha above

Reply:
I wanted to write that probably we don't need these anymore but wanted to leave that to you to delete them later :)

Reply:
i will create an issue for this, sounds good